src/core/app_backend.tf

```
@@ -522,6 +525,11 @@ data "azurerm_key_vault_secret" "app_backend_IOLOGIN_TEST_USERS" {
   key_vault_id = module.key_vault_common.id
 }
 
+data "azurerm_key_vault_secret" "app_backend_UNIQUE_EMAIL_ENFORCEMENT_USER" {
+  name         = "appbackend-UNIQUE_EMAIL_ENFORCEMENT_USER"
```

Secret names can only contain alphanumeric characters and dashes

```suggestion
  name         = "appbackend-UNIQUE-EMAIL-ENFORCEMENT-USER"
```